index.html

```diff
@@ -385,10 +397,10 @@ <h2><dfn>observe()</dfn> method</h2>
             <a>entryTypes</a> sequence.
             </li>
             <li>Remove all types from <var>entry types</var> that are not
-            contained in <a>supported entry types</a>. The user agent SHOULD
-            notify developers if <var>entry types</var> is modified. For
-            example, a console warning listing removed types might be
-            appropriate.</li>
+            contained in the <a>relevant supported entry types</a>. The user
```

Comment:
Two issues here:

1) This is not well-defined, unfortunately.  "relevant supported entry types" is defined in terms of "relevant global object", but without defining _what_ it's a relevant global object of.  "relevant global object" is defined for platform objects, not as an abstract concept.  Concretely, the definition of "relevant supported entry types" should include an input: the platform object it's for.  And then it can pass that input through to the "relevant global object" concept.  So in this case you'd say "the relevant supported entry types of _observer_" and the definition of the "relevant supported entry types" would talk about the "relevant supported entry types for a platform object _o_ are ... of the relevant global object of _o_".

2) The definition of "contained in" is not clear here, given that the "relevant supported entry types" is a JS Array.  How is that determination made?  Is the method observable via side-effects on the page?  Ideally we would sidestep this by storing both an infra list and the JS Array representation of it on the global and just consult the list here.

Reply:
Improved the definition and added the infra list.